contracts/BSCValidatorSet.sol

```diff
@@ -413,6 +435,16 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       require(newBurnRatio <= BURN_RATIO_SCALE, "the burnRatio must be no greater than 10000");
       burnRatio = newBurnRatio;
       burnRatioInitialized = true;
+    } else if (Memory.compareStrings(key, "maxNumOfMaintaining")) {
+      require(value.length == 32, "length of maxNumOfMaintaining mismatch");
+      uint256 newMaxNumOfMaintaining = BytesToTypes.bytesToUint256(32, value);
```

Comment:
Can we have an initial value for `maxNumOfMaintaining` and `maintainSlashScale`, just like the bruno burn?

Reply:
Also please add some testcase to change the value of these.

Reply:
Ok,   initial values added